mne/decoding/time_gen.py

```diff
@@ -1508,3 +1516,14 @@ def _set_cv(cv, clf=None, X=None, y=None):
         raise ValueError('Some folds do not have any train epochs.')
 
     return cv, cv_splits
+
+
+SCORERS = dict(r2=sklearn.metrics.r2_score,
```

Comment:
I think sklearn has this sort of function already. If I'm correct, it'd be better to use theirs than recreating it here to avoid having to support it.